SPECS/perl-CPAN-Meta-Check/perl-CPAN-Meta-Check.spec

```diff
@@ -0,0 +1,171 @@
+Name:		perl-CPAN-Meta-Check
+Summary:	Verify requirements in a CPAN::Meta object
+Version:	0.014
+Release:	12%{?dist}
+License:	GPL+ OR Artistic
```

Comment:
>	 [](http://example.com/codeflow?start=8&length=1)

Nit: Can you align the tags in this section?

Reply:
Fixed alignment.